features_module_how_to_share_sites_config_with_oth.md

```diff
@@ -0,0 +1,118 @@
+# Features module: how to share site’s config with others?
+
+If you already had a chance to build one or two Drupal sites,  you probably know Drupal site consists of three parts:
```

Comment:
That's why I asked you ... for me it all looks good, I'm totally blind when it comes to graphical representation ;-)

@OritGeron I need your drawing skills again :-) can you please?